app/main/views/suppliers.py

```diff
@@ -92,6 +92,20 @@ def delete_supplier(code):
     return jsonify(message="done"), 200
 
 
+@main.route('/suppliers', methods=['DELETE'])
+def delete_suppliers():
+    try:
+        db.session.query(Supplier).delete()
+        db.session.commit()
```

Comment:
You should clean out the Elasticsearch index, too.  The stupid and easy way is to just use

delete_indices()
create_indices()

from app/search_indices.py.  (There's already an import from that file at the top.)  If you look at the source of that file, you'll see how to do it the smarter way.